ThirdPartyNotices.txt

```diff
@@ -0,0 +1,54 @@
+THIRD PARTY NOTICES
+
+This repository incorporates material as listed below or described in the code.
+
+------------------------------------------------------------
+https://github.com/microsoft/Nova
+
+Licensed under MIT
+
+MIT License
+
+Copyright (c) Microsoft Corporation.
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE
+
+------------------------------------------------------------
+https://github.com/espressosystems/jellyfish
+https://github.com/han0110/plonkish/
+https://github.com/espressosystems/hyperplonk
+
+Licensed under MIT
+
+MIT License
+
+Copyright (c) 2022 Espresso Systems, han0110
```

Comment:
This branch technically contains none of these but eh, the discrepancy should self correct soon enough.